capabilities/logging-tracing-monitoring.md

```diff
@@ -0,0 +1,37 @@
+---
+displayName: Logging-Tracing-Monitoring
+epicsLabels:
+  - area/logging
+  - area/tracing
+  - area/monitoring
+---
+
+ ## Scope
+Enable the operator and developer to easily observe the state of the Kyma cluster and distributed applications running on Kyma. The pre-bundled infrastructure exposes and collects the data through application logs and metrics as well as transaction traces. This infrastructure integrates with the cloud provider-specific tools.
+
+Ensuring that Kyma components create application logs, metrics, and trace data in a consistent way and on a sufficient detail level is not in scope of this capability.
```

Comment:
I am not sure if it should be outside. At least in case of tracing consistency is essential. Cannot imagine if every component uses different standards. In my opinion, the scope should contain all the guidelines and the unifications. It would be nice if at least kyma components use the same logging levels and the same configuration mechanism for changing it.

Reply:
So I removed the section and keep it unspecific. Let's see at a later time who at the end will drive the contribution guidelines itself and who will check that they are followed